js/superscripts.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+// Update superscripts to not suffer misinterpretation when copied and pasted as plain text.
+// For example,
+// * Replace `10<sup>3</sup>` with `10³` so it gets pasted as that rather than `103`.
+// * Replace `10<sup>-<var>x</var></sup>` with
+//   `10<span aria-hidden="true"> ↑ </span><sup>-<var>x</var></sup>`
+//   so it gets pasted as `10 ↑ -x` rather than `10-x`.
+// * Replace `2<sup><var>a</var> + 1</sup>` with
+//   `2<span …> ↑ (</span><sup><var>a</var> + 1</sup><span …>)</span>`
+//   so it gets pasted as `2 ↑ (a + 1)` rather than `2a + 1`.
+
+const supReplacements = new Map([
+  ['0', '⁰'],
+  ['1', '¹'],
+  ['2', '²'],
+  ['3', '³'],
+  ['4', '⁴'],
+  ['5', '⁵'],
+  ['6', '⁶'],
+  ['7', '⁷'],
+  ['8', '⁸'],
+  ['9', '⁹'],
+  ['(', '⁽'],
+  [')', '⁾'],
+  ['=', '⁼'],
+  ['+', '⁺'],
+  ['-', '⁻'],
```

Comment:
There are visual equivalents for multiplication and division as well:
```suggestion
  ['-', '⁻'],
  ['×', 'ˣ'],
  ['/', 'ᐟ'],
```

though I suppose it's considered poor form to abuse Unicode in this way (https://github.com/tc39/ecmarkup/pull/265#issuecomment-722978302).

(Those are U+02e3 "Modifier Letter Small X" and U+141F "Canadian Syllabics Final Acute" respectively.)

Why does Unicode have superscript addition but not multiplication, anyway?

Reply:
:shrug:

Reply:
> There are visual equivalents for multiplication and division as well:
> 
> though I suppose it's considered poor form to abuse Unicode in this way ([#265 (comment)](https://github.com/tc39/ecmarkup/pull/265#issuecomment-722978302)).
> 
> (Those are U+02e3 "Modifier Letter Small X" and U+141F "Canadian Syllabics Final Acute" respectively.)

Yeah, I definitely don't want to misuse characters like that—preserving `10<sup>1 × 3</sup>` is better than replacing it with `10¹ ˣ ³`. If the inconsistency between `+` and `×` bothers you, I'd rather just remove the former (but note that use as addition in variable-free text is unexpected; it's included here to cover sign assertions such as `10<sup>+3</sup>` → `10⁺³`, which don't currently exist in ECMA-262 or ECMA-402).

> Why does Unicode have superscript addition but not multiplication, anyway?

:shrug: